projects/storefrontlib/shared/services/import-export/file-download.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { ExportFileOptions } from '../../models/file';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class FileDownloadService {
+  download(fileContent: string, fileOptions: ExportFileOptions): void {
+    const { fileName, type, extension } = fileOptions;
+    const blob = new Blob(['\ufeff' + fileContent], {
```

Comment:
1. Do we have to put the BOM `\ufeff` for every downloaded file? I guess no.
2. Does this BOM apply for every possible charset encoding? I guess no.
Then... which service should be responsible for adding this BOM and when?